news/_posts/2021-09-08-content-style-guide.md

```diff
@@ -0,0 +1,55 @@
+
+![Content style guide banner](/assets/images/2021/09/build-content-style-guide-banner.png)
+
+### A reference document for writing content
+
+When designing a new feature, contributors to the PrestaShop project have to think about the copy: feature labels, help texts, error messages, etc. Because there are many contributors to the project, multiple writers take action on the content by adding new [strings](https://zeroheight.com/80a6c5a61/v/latest/p/36889f-glossary/b/34bc6f). Moreover, the community is very diverse, and every writer has his/her own writing style. To ensure consistency, it was important to provide a document that anyone writing for the project could refer to. 
+
+This reference document sets out principles and guidelines for the writing and formatting of text on PrestaShop software. So, for example, if you are designing a modal, a button, or writing an error message, you'll find what you are looking for in the Content style guide!
+
+This guide also contains a glossary that will help you understand the vocabulary used in the PrestaShop project.
+
+[Read Content style guide](https://zeroheight.com/80a6c5a61/v/latest/p/2971bf-content-style-guide){: .btn .btn-lg .btn-info}
+{: .text-center}
+
+### A big challenge but many benefits
+
+Let's face it, there's a lot of work to be done to harmonize and improve the wording on PrestaShop software. But these content guidelines are a good place to start!
+
+The Content style guide is expected to have many benefits: 
+
+- For contributors, the guidelines will help them better understand how the wording reviews work and verify that the strings in their pull requests meet the standards.
+- For both maintainers and contributors, content guidelines will save time in the review process by avoiding back and forth before a wording contribution is merged into the project.
+- For merchants, consistent, clear, and useful messages will improve their experience on PrestaShop software.
+
+It's a win-win!
+
+![Thumb up](/assets/images/2021/09/build-thumb-up.png)
+
+### Next steps 
+
+From now on, wording reviews will be based on the guidelines in the guide. Any new string added to the project will have to meet the standards. So, if your pull request contains new strings, please use the Content style guide as a reference. 
+
+Regarding existing strings, they'll have to be modified too. That's a lot of work. But together we are stronger! :muscle: If you think that a string could be improved, feel free to [open an issue](https://github.com/PrestaShop/PrestaShop/issues/new/choose) (and why not a pull request?) and make a suggestion based on the guidelines in the guide. 
+
+Also, any suggestions for improvement are welcomed! Nothing is cast in stone. If you think a component is missing, or if you can't find the answer to your question in the guide, you can also open an issue.
```

Comment:
On which github repository should people open issues?

Reply:
@matks At first we wanted to create a specific repo, but Pablo said that it was better to use the main repository for now. If we notice that there are a lot of issues and/or PRs we might consider creating a new repo.